tree/src/main/java/me/coley/cafedude/tree/Constant.java

```diff
@@ -194,8 +194,8 @@ public static Constant of(@Nonnull Descriptor value) {
 
 	/**
 	 * Create a constant from a {@link Handle} value.
-	 * <p>
-	 * Corresponds to {@code CpMethodHandle}.
+	 *
+	 * <p>Corresponds to {@code CpMethodHandle}.
```

Comment:
Did your auto-formatter move all the p blocks to the same line?

Reply:
That is what the checkstyle requests

Reply:
newline before <p> and <p> at start of line

Reply:
Eww, we can change the checkstyle format for that, because it looks silly.

In Javadocs `<p>` is used similarly to `<br>` but for complete paragraphs. This conflicts with that idea IMO.